index.html

```diff
@@ -5,92 +5,161 @@
 	<title>Home</title>
 	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
 	<link rel="stylesheet" href="css/index.css">
+	
+	
 	<!--[if IE]>
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
 
-<body>
-
-	<h1>You got this!  Good luck.</h1>
 
-<!-- Menu Content 
-S&J
 
-Home
-Services
-Contact
--->
-
-
-<!-- Home Page Content:
+	<body>
+		<div class="container home">
+			<!-- Menu Content -->
+			<nav class='home-nav'>
+				<h1>S&J</h1>
+				<div class="menu">
+					<div class="menu-items">
+						<a href="index.html">Home</a>
```

Comment:
Is this div necessary? I think you could get the same functionality with just the outer div.